host-configs/Stanford/sherlock-gcc10-ompi4.1.2-openblas0.3.10-cuda11.5.0-sm80.cmake

```diff
@@ -0,0 +1,66 @@
+set(CONFIG_NAME "sherlock-gcc10-ompi4.1.2-openblas0.3.10-cuda11.5.0-sm80" CACHE PATH "")
+
+set(SOFTWARE_ROOT /share/software/user/open CACHE PATH "")
+
+set(GCC_ROOT "${SOFTWARE_ROOT}/gcc/10.1.0" CACHE PATH "")
+set(MPI_ROOT "${SOFTWARE_ROOT}/openmpi/4.1.2" CACHE PATH "")
+set(CUDA_HOME "${SOFTWARE_ROOT}/cuda/11.5.0" CACHE PATH "")
+set(CMAKE_CUDA_ARCHITECTURES "80" CACHE STRING "")
+set(CUDA_ARCH "sm_80" CACHE STRING "")
+set(CUDA_TOOLKIT_ROOT_DIR "${CUDA_HOME}" CACHE STRING "")
+
+site_name(HOST_NAME)
+
+# Compilers
+set(CMAKE_C_COMPILER       "${GCC_ROOT}/bin/gcc"      CACHE PATH "")
+set(CMAKE_CXX_COMPILER     "${GCC_ROOT}/bin/g++"      CACHE PATH "")
+set(CMAKE_Fortran_COMPILER "${GCC_ROOT}/bin/gfortran" CACHE PATH "")
+
+# OpenMP options
+set(ENABLE_OPENMP OFF CACHE BOOL "")
+set(MPI_C_COMPILER       "${MPI_ROOT}/bin/mpicc"   CACHE PATH "")
+set(MPI_CXX_COMPILER     "${MPI_ROOT}/bin/mpic++"  CACHE PATH "")
+set(MPI_Fortran_COMPILER "${MPI_ROOT}/bin/mpifort" CACHE PATH "")
+set(MPIEXEC              "${MPI_ROOT}/bin/mpirun"  CACHE PATH "")
+# MPI options
+set(ENABLE_MPI ON CACHE PATH "" FORCE)
+
+set(MPIEXEC_NUMPROC_FLAG "-n" CACHE STRING "")
+set(ENABLE_WRAP_ALL_TESTS_WITH_MPIEXEC ON CACHE BOOL "")
+
+# CUDA options
+#set(ENABLE_CUDA OFF)
+set(ENABLE_CUDA ON CACHE BOOL "" FORCE)
```

Comment:
You could surely have one unique cmake file by checking `CUDA` with some `if (ENABLE_CUDA)...`.
Because I think that all the rest remains unchanged?

Reply:
Perfectly true. It was done this way before. I'll refactor as soon as I have the GPU not reaching time out. :thinking: